filter/store.go

```diff
@@ -29,6 +31,20 @@ type FilterOuput struct {
 	InstanceID string     `json:"instance_id"`
 	State      string     `json:"state,omitempty"`
 	Downloads  *Downloads `json:"downloads,omitempty"`
+	Events     Events     `json:"events,omitempty"`
+}
+
+// Events represents a list of array objects containing event information against the filter job
+type Events struct {
+	Error []EventItem `json:"error,omitempty"`
```

Comment:
Not sure we should have separation between error and info events. Maybe just leave Events as an []EventItem, and the type would allow us any details we might want around error or info?

Reply:
This is what the filter-api expects. I added a comment in a trello card that the import api and filter-api are different when handling events.